src/Pagination/PaginationArrow.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FunctionComponent } from 'react'
+import { PreviousArrow, NextArrow } from './arrows'
+import clsx from 'clsx'
+
+import styles from './pagination.module.scss'
+
+interface Props {
+  arrowType: 'Previous' | 'Next'
+  className?: string
+  [rest: string]: unknown // ...rest property
+}
+
+const PaginationArrow: FunctionComponent<Props> = ({ arrowType, className, ...rest }) => {
+  let children
+  let ariaLabel
+  if (arrowType === 'Previous') {
+    ariaLabel = 'Previous'
+    children = <PreviousArrow />
+  }
+  if (arrowType === 'Next') {
+    ariaLabel = 'Next'
+    children = <NextArrow />
+  }
+  return (
+    <button aria-label={ariaLabel} className={clsx(styles.arrowButton, className)} {...rest}>
```

Comment:
why don't you just use `arrowType` instead of `ariaLabel`? `<button aria-label={arrowType} ...>`

It accomplishes the same thing

Reply:
Ahh good point. The arrowType was a last minute change to make it a little more simple.